lib/buildTypes/minifyJS.js

```diff
@@ -1,5 +1,4 @@
 var UglifyJS = require("uglify-js");
-var removeSourceMapUrl = require("../remove_source_map_url");
```

Comment:
yes.

i think that uglify will also strip the sourceMap https://github.com/mishoo/UglifyJS2/blob/41a9329409ca0f1273b6ccdcc9770713f44ec55a/test/mocha/input-sourcemaps.js